### PR TITLE
User Profile Implementation - Can get Current User in Memory Game

### PR DESCRIPTION
**Pull request by YSJL**

### CODE REVIEW

File: app/src/main/java/com/example/k_dev_master/MainActivity.java [hunk 1]
```diff
@@ -77,6 +119,7 @@ public void onClick(View v) {
                     startActivity(intent);
                 } else if (gameSelect == 2) {
                     Intent intent = new Intent(getApplicationContext(), MemoryGame.class);
+                    intent.putExtra("curUser", curUser);
```
Comment (YSJL) [inline app/src/main/java/com/example/k_dev_master/MainActivity.java:122]: MainActivity passes curUser to MemoryGame

File: app/src/main/java/com/example/k_dev_master/memorygame/MemoryGame.java [hunk 2]
```diff
@@ -66,8 +72,14 @@ public class MemoryGame extends AppCompatActivity {
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         getSupportActionBar().hide();
         super.onCreate(savedInstanceState);
+
+        if (getIntent().getStringExtra("curUser").length() != 0) {
+            curUser = getIntent().getStringExtra("curUser");
+            System.out.println(curUser);
```
Comment (YSJL) [inline app/src/main/java/com/example/k_dev_master/memorygame/MemoryGame.java:78]: Memory Game gets curUser here

**Comment by schae44** [comment]
Confirmed!

**Comment by gosbhs12** [comment]
Nice work!
can you merge from Sarah's and remerge to me?

**Pull request merged**